<launch>
	<arg name="name" default="ros_object_detector_server" />
	<arg name="tran_path" default="/home/wjlee/catkin_ws/src/ros_object_detector" />

	<arg name="test_image_path_for_object" default="cutter_detection/images/m_1.jpg" />	
	<arg name="dnn_config_path_for_object" default="cutter_detection/yolov4-custom.cfg" />
	<arg name="dnn_weight_path_for_object" default="cutter_detection/backup/yolov4-custom_best.weights" />
	<arg name="dnn_meta_path_for_object" default="cutter_detection/obj.data" />
	
	<node pkg="ros_object_detector" type="ros_object_detector_server" respawn="false" output="screen" name="$(arg name)" >

		<!-- ORANGE -->
		
		<param name="test_image_path_for_object" type="string" value="$(arg tran_path)/$(arg test_image_path_for_object)" />
		<!-- DNN Config File Path -->

		<param name="dnn_config_path_for_object" type="string" value="$(arg tran_path)/$(arg dnn_config_path_for_object)" />
		<!-- DNN Weight File Path -->
		<param name="dnn_weight_path_for_object" type="string" value="$(arg tran_path)/$(arg dnn_weight_path_for_object)" />
		<!-- DNN meta File Path -->
		<param name="dnn_meta_path_for_object" type="string" value="$(arg tran_path)/$(arg dnn_meta_path_for_object)" />
		<!-- DNN meta File Path -->

		<!-- internal name -->
		<param name="name" value="$(arg name)" />
	</node>
</launch>
